engine/clients/redis/search.py

```diff
@@ -41,16 +42,16 @@ def init_client(cls, host, distance, connection_params: dict, search_params: dic
         cls._ft = cls.conns[random.randint(0, len(cls.conns)) - 1].ft()
```

Comment:
`cls.conns`, `cls._is_cluster`, `cls._ft`, `cls.knn_conditions` require definitions on class level (values are not required, only type hints)